docs/tutorials/13_quantum_random_access_optimizer.ipynb

```diff
@@ -0,0 +1,854 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "The objective function value: 3.0\n",
+      "x: [0 0 0 1 0 0]\n",
+      "relaxed function value: -8.999997502617678\n",
+      "The number of distinct samples is 1.\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Round the relaxed solution using semi-deterministic rounding\n",
+    "semidterministic_rounding = SemideterministicRounding()\n",
+    "sdr_results = semidterministic_rounding.round(rounding_context)\n",
+    "qrao_results_sdr = qrao.process_result(\n",
+    "    problem=problem, encoding=encoding, relaxed_result=relaxed_results, rounding_result=sdr_results\n",
+    ")\n",
+    "\n",
+    "print(\n",
+    "    f\"The objective function value: {qrao_results_sdr.fval}\\n\"\n",
+    "    f\"x: {qrao_results_sdr.x}\\n\"\n",
+    "    f\"relaxed function value: {-1 * qrao_results_sdr.relaxed_fval}\\n\"\n",
+    "    f\"The number of distinct samples is {len(qrao_results_sdr.samples)}.\"\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "The objective function value: 9.0\n",
+      "x: [1 0 1 0 0 1]\n",
+      "relaxed function value: -8.999997502617678\n",
+      "The number of distinct samples is 56.\n"
+     ]
+    }
+   ],
+   "source": [
+    "magic_rounding = MagicRounding(sampler=sampler)\n",
+    "mr_results = magic_rounding.round(rounding_context)\n",
+    "qrao_results_mr = qrao.process_result(\n",
+    "    problem=problem, encoding=encoding, relaxed_result=relaxed_results, rounding_result=mr_results\n",
+    ")\n",
+    "\n",
+    "print(\n",
+    "    f\"The objective function value: {qrao_results_mr.fval}\\n\"\n",
+    "    f\"x: {qrao_results_mr.x}\\n\"\n",
+    "    f\"relaxed function value: {-1 * qrao_results_mr.relaxed_fval}\\n\"\n",
+    "    f\"The number of distinct samples is {len(qrao_results_mr.samples)}.\"\n",
+    ")"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Appendix\n",
+    "### How to verify correctness of your encoding\n",
+    "We assume for sake of the QRAO method that **the relaxation commutes with the objective function.** This notebook demonstrates how one can verify this for any problem (a `QuadraticProgram` in the language of Qiskit Optimization). One might want to verify this for pedagogical purposes, or as a sanity check when investigating unexpected behavior with the QRAO. Any problem that does not commute should be considered a bug, and if such a problem is discovered, we encourage that you submit it as [an issue on GitHub](https://github.com/Qiskit/qiskit-optimization/issues).\n",
+    "\n",
```

Comment:
Assuming we want to keep the text here about bug submission the link should at least be update to the new community repo location.